ajax/libs/isomorphic-fetch/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "isomorphic-fetch",
+  "description": "Isomorphic WHATWG Fetch API, for Node & Browserify",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthew-andrews/isomorphic-fetch.git"
+  },
+  "author": "Matt Andrews <matt@mattandre.ws> (http://mattandre.ws/)",
+  "license": "MIT",
+  "filename": "fetch-npm-node.js",
+  "keywords": [
+    "fetch",
+    "api",
+    "node",
+    "browserify"
+  ],
+  "npmName": "isomorphic-fetch",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "fetch-*"
```

Comment:
@pvnr0082t Why? Because `fetch-npm-node.js` is listed as main file in [package.json](https://github.com/matthew-andrews/isomorphic-fetch/blob/master/package.json).